messages/remove.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+	"commandDescription": "[Short description]",
```

Comment:
Can we add the messages right away? We wouldn't have to remember to revisit later, especially now that our release cycle is kind of spontaneous.

Reply:
Yeah, I forgot to do that. I'll get on it ASAP.